manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/processor/PluginProcessor.java

```diff
@@ -109,9 +109,12 @@
 
         Map<ManifestMeta,String> manifestMetaVsSnippets = customSnippetsProvider.fetchUnmergedCustomSnippets();
         if(manifestMetaVsSnippets != null && !manifestMetaVsSnippets.isEmpty()){
+            Set<String> kinds = new HashSet<>();
             manifestMetaVsSnippets.forEach((manifestMeta, snippet)->{
-                WorkflowLogger.warn(ManifestGeneratorActivity.IGNORING_CUSTOM_SNIPPET,manifestMeta.getKind());
+                kinds.add(manifestMeta.getKind());
             });
```

Comment:
Can be combined into a single operation.
kinds = manifestMetaVsSnippets.keySet().stream().map(ManifestMeta::getKind).collect(Collectors.toSet());